storage/sql/migrations/updates/20190204170400_create_transfer_data_field_in_trs_table.sql

```diff
@@ -12,4 +12,11 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-INSERT INTO "votes" ("transactionId", "votes") VALUES ${values:raw};
+
+/*
+  DESCRIPTION: Add transfer data column to trs.
+
+  PARAMETERS: None
+*/
+
+ALTER TABLE "trs" ADD COLUMN IF NOT EXISTS "transferData" BYTEA;
```

Comment:
Migrations must be fully atomic, all the changes related to one particular thing must be in one migration file. We run single migration file in SQL transaction, so if something failed it should failed as a group. 

In this case we should do this steps in single migration file.  

1. We should create a new column "transferData"
2. Move old transfer data to new column 
3. Remove transfer table 

So as of now if that migration file run, we will be done fully what was required.